core/templates/pages/about-page/about-page.directive.html

```diff
@@ -7,20 +7,20 @@ <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
     <div class="oppia-page-card oppia-static-content">
       <div class="about-nav">
         <ul class="oppia-about-tabs">
-          <li class="oppia-about-tabs-active">
+          <li ng-class="{ 'oppia-about-tabs-active': $ctrl.activeTabName === $ctrl.TAB_ID_ABOUT }">
```

Comment:
Here and below: drop the space after the { and before the }

Reply:
Done.